bin/tectonics-modeling-L0/session1-aspect.ipynb

```diff
@@ -0,0 +1,2840 @@
+{
```

Comment:
<p>I am a bit confused in the subsequent section if the user will compute the Ra-Nu Relationship or not. If not, maybe we can remove the related sentences and make it an exercise later in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>